greedybear/cronjobs/db_backup.py

```diff
@@ -0,0 +1,45 @@
+import os
+from datetime import datetime
+
+from cStringIO import StringIO
+from django.core import management
+
+from greedybear import settings
+from greedybear.apps import GreedyBearConfig
+from greedybear.cronjobs.base import Cronjob
+
+
+class DbBackup(Cronjob):
+    def __init__(self):
+        super().__init__()
+        self.interval = settings.DB_BACKUP_FREQUENCY
+        self.location = (
+            str(settings.DB_BACKUP_STORAGE_OPTIONS["location"]) or "/opt/backups"
+        )
+        self.max_backups = int(settings.DB_BACKUP_STORAGE_OPTIONS["max_backups"]) or 10
+
+    @property
+    def minutes_back_to_lookup(self):
+        return self.interval
+
+    def _remove_oldest_backup(self, dir):
+        # removes files older than self.max_backups
+        files = os.listdir(dir)
+        if len(files) >= self.max_backups:
+            oldest_file = min(files, key=os.path.getctime)
+            os.remove(os.path.abspath(oldest_file))
+
+    def run(self):
+        self._remove_oldest_backup(self.location)
+        app_name = GreedyBearConfig.name
+        backup_file = (
+            f"{self.location}/db_backup_{datetime.now().strftime('%Y%m%d-%H%M%S')}.json"
+        )
+        # Init a new buffer
+        buf = StringIO()
+        # Call command with and pipe stdout to buffer
+        management.call_command("dbbackup", app_name, stdout=buf)
+        buf.seek(0)
+        # Write buffer to file
+        with open(backup_file, "w", "utf-8") as dump:
+            dump.write(buf.read())
```

Comment:
can you show an example of a successful run of this cron?

plus, there are no logs in all the cron, it is impossible to debug what is happening without them